custom/enikshay/two_b_release_1/management/commands/enikshay_2b_case_properties.py

```diff
@@ -317,20 +317,21 @@ def migrate_occurrence(self, occurrence, episodes):
             },
         )
 
-    def migrate_episode(self, episode, episodes):
-        self.total_episodes += 0
-
+    @staticmethod
+    def _get_last_episode_id(indices, episodes):
         def is_episode_of_occurrence(episode, occurrence_id):
             return any(index.referenced_id == occurrence_id for index in episode.indices)
 
-        occurrence_ids = [index.referenced_id for index in episode.indices
+        occurrence_ids = [index.referenced_id for index in indices
                           if index.referenced_type == CASE_TYPE_OCCURRENCE]
         occurrence_id = occurrence_ids[0] if occurrence_ids else None
+        return max((case.opened_on, case.case_id) for case in episodes
+                   if is_episode_of_occurrence(case, occurrence_id) and not case.closed)[1]
```

Comment:
@esoergel 
For this situation, I don't think we need to filter on opened cases (we just need the last opened episode)